src/solana/utils/security_txt.py

```diff
@@ -0,0 +1,64 @@
+"""Utils for security.txt."""
+
+HEADER = "=======BEGIN SECURITY.TXT V1=======\0"
+FOOTER = "=======END SECURITY.TXT V1=======\0"
+
+VALID_KEYS = [
+    "name",
+    "project_url",
+    "contacts",
+    "policy",
+    "preferred_languages",
+    "source_code",
+    "encryption",
+    "auditors",
+    "acknowledgements",
+    "expiry",
+]
+
+REQUIRED_KEYS = ["name", "project_url", "contacts", "policy"]
+
+
+def parse(data: bytes):
```

Comment:
Can we define the return type?

Reply:
Sounds good! I was waiting on further suggestions.

Reply:
Will do!

Reply:
We know what fields the returned dictionary has right? If so, it can be a dataclass instead of a dict

Reply:
The fields/keys are fixed as per security.txt specs.
 But, they do have few mandatory fields and the rest are optional.

I will shift this to a data class.

Reply:
Yes, please replace the asserts with `raise`